api/v1alpha1/release_types.go

```diff
@@ -44,6 +44,13 @@ const (
 	// releaseConditionType is the type used when setting a release status condition
 	releaseConditionType string = "Succeeded"
 
+	// AutoReleaseLabel is the label name for the auto-release setting
+	AutoReleaseLabel = "release.appstudio.openshift.io/auto-release"
+
+	// BindingDeploymentStatusConditionType is the condition type to retrieve from the ComponentDeploymentConditions
+	// in the SnapshotEnvironmentBinding's status to copy into the Release status
+	BindingDeploymentStatusConditionType string = "AllComponentsDeployed"
```

Comment:
This will probably change in the future but leave the AutoReleaseLabel were it was. The content of this block is used for conditions exclusively.

Reply:
Why? It isn't commented that in the constant blocks. There are already comments for each constant saying what it is for. Why would we want 
```
const (
 some constants
)
const (
 some more constants
)
```
in the same file?

Reply:
Way more clear to have a block just for condition. Also, this particular change is unrelated and should not be part of this change.

Reply:
Disagree on way clearer, but agree it is an unrelated change so I will remove it